staging/src/k8s.io/code-generator/cmd/conversion-gen/args/args.go

```diff
@@ -68,13 +68,13 @@ func NewDefaults() (*args.GeneratorArgs, *CustomArgs) {
 
 // AddFlags add the generator flags to the flag set.
 func (ca *CustomArgs) AddFlags(fs *pflag.FlagSet) {
-	pflag.CommandLine.StringSliceVar(&ca.BasePeerDirs, "base-peer-dirs", ca.BasePeerDirs,
+	fs.StringSliceVar(&ca.BasePeerDirs, "base-peer-dirs", ca.BasePeerDirs,
```

Comment:
You could send fixes like this as a PR of its own

Reply:
This is, in fact, crucial to the implementation. Well, the tests of the implementation. 

I'm calling the specific generator function from the main binary. Without this, the tests fail, as I'm unable to pass proper flagset here.

What I can do, is to send this fix earlier, to be merged before this refactor, in a separate PR.